tool_collections/cufflinks/cuffcompare/cuffcompare_wrapper.xml

```diff
@@ -229,8 +229,10 @@ If you ran cuffcompare with the -r option, tracking rows will contain the follow
   6                 r               Repeat. Currently determined by looking at the reference sequence and applied to transcripts where at least 50% of the bases are lower case        
   7                 p               Possible polymerase run-on fragment        
   8                 u               Unknown, intergenic transcript        
-  9                 o               Unknown, generic overlap with reference        
-  10             .               (.tracking file only, indicates multiple classifications)
+  9                 o               Unknown, generic overlap with reference 
```

Comment:
6, 8 and 9 do not match the documentation at http://cole-trapnell-lab.github.io/cufflinks/cuffcompare/index.html